lib/index.js

```diff
@@ -29,12 +29,41 @@ function defaultHandler(level, error) {
 	const message = JSON.stringify({
 		_level: level,
 		_timestamp: new Date().toISOString(),
-		_commit_hash: config.gitCommit,
+		_commit_hash: config.commitHash,
 		_service: config.service,
 		...formattedError,
 	});
 
-	process.stderr.write(message + '\n');
+	write(level, `${message}\n`, error);
+}
+
+function write(level, message, error) {
+	if (global.window && global.window.console) {
```

Comment:
Can this go outside of the function scope, it shouldn't need to be called every write operation?

Reply:
Indeed. I moved it into here when testing but forgot to move it back, cheers.

Reply:
Done in https://github.com/cuvva/cuvva-log-node/pull/2/commits/99a47637de9f29a6b6ab0bb4bbbe5b54de14ec61